test/guit/fetch.t

```diff
@@ -2,11 +2,11 @@ Tests fetch command
   $ mkdir simple
   $ mkdir simple-clone
   $ cd simple
+  $ git init 2> /dev/null
+  Initialized empty Git repository in $TESTCASE_ROOT/simple/.git/
   $ git config init.defaultBranch master
```

Comment:
That line is not necessary anymore, then?

Reply:
Hmmhmm, it seems required and even if we do `&> /dev/null`, `git init` in `cram` still prints something...